zds/tutorialv2/models/database.py

```diff
@@ -685,13 +710,24 @@ class Meta:
     _meta_description = None
     _manifest = None
 
+    def save(self, *args, **kwargs):
+        super().save(*args, **kwargs)
+        if hasattr(self, "_authors"):
+            self._authors = list(self.authors.all())
```

Comment:
Je suggère ceci pour que la requête ne soit faite en base de données que si `public_authors()` est appelée plus loin dans le code.
```suggestion
            delattr(self, "_authors")
```